crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -400,9 +400,10 @@ def services
 
       def openstackbackup
         crowbar_lib_dir = "/var/lib/crowbar"
-        dump_path = "#{crowbar_lib_dir}/upgrade/6-to-7-openstack_dump.sql"
+        dump_path = "#{crowbar_lib_dir}/backup/6-to-7-openstack_dump.sql.tar.gz"
         if File.exist?(dump_path)
           Rails.logger.debug("OpenStack backup already exists. Skipping...")
+          ::Crowbar::UpgradeStatus.new.end_step
```

Comment:
I'm not sure if this is the right logic. Silently skipping the backup if the file already exists? I'd rather make it like the crowbar backup that the filename is passed from the UI and contains some timestamp so that there can be more than one without conflict.